packages/server/README.md

```diff
@@ -20,21 +20,49 @@ This will start the server, which will listen on port **22300** on **localhost**
 ## Setup the database
 
 You can setup the container to either use an existing PostgreSQL server, or connect it to a new one using docker-compose
-
 ### Using an existing PostgreSQL server
 
-To use an existing PostgresSQL server, set the following environment variables in the .env file:
+To use an existing PostgresSQL server you must configure some environment variables in the .env file: 
 
+First set the `DB_CLIENT` environment variable to `pg`
 ```conf
 DB_CLIENT=pg
+```
+
+#### Defining the connection
+Use a connection string variable:
+```conf
+POSTGRES_CONNECTION_STRING=postgresql://username:password@your_joplin_postgres_server:5432/joplin
+```
+Or use individual variables:
+```conf
 POSTGRES_PASSWORD=joplin
 POSTGRES_DATABASE=joplin
 POSTGRES_USER=joplin
 POSTGRES_PORT=5432
 POSTGRES_HOST=localhost
 ```
 
-Make sure that the provided database and user exist as the server will not create them.
+*Make sure that the provided database and user exist as the server will not create them.*
+
+#### SSL Options
```

Comment:
I always wonder - since SSL was deprecated 10 years ago and is not used anymore, why do we keep using the term "SSL"? Shouldn't it be TLS? But on the other hand I see that for example the Postgres doc refers to SSL too, so I'm not sure what's best these days.

Reply:
I take your point, the props need to be call SSL I think for node-postgres, but I will look to add something the the read me to address your point.

Reply:
Actually I was just asking, I just wonder and found some answers [here](https://expeditedsecurity.com/blog/ssl-or-tls/). So based on this and the fact that the Postgres doc refers to it as SSL, let's use SSL here too after all.

Reply:
Thanks for the link to that blog. I like when docs explain things so I will reflect this.